src/guide/frontend/android.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Android Client SDK
+---
+
+# Android Client SDK
+
+The Passwordless.dev Android client SDK gives users the ability to leverage their device’s built-in fingerprint sensor and/or FIDO security keys for secure passwordless access to websites and native applications that support the FIDO2 protocols
+
+## Requirements
+
+- Android 9.0 (API level 28) or higher
+- Java 8 or higher
+- [Having completed the 'Get started' guide](../get-started.md).
+
+## Installation
+
+::: tabs
+
+@tab Apache Maven
+
+```xml
+<dependency>
+  <groupId>com.bitwarden</groupId>
+  <artifactId>passwordless-android</artifactId>
+  <version>1.0.1</version>
+</dependency>
+```
+
+@tab Gradle Kotlin DSL
+
+```kotlin
+implementation("com.bitwarden:passwordless-android:1.0.1")
+```
+
+@tab Gradle Groovy DSL
+
+```groovy
+implementation 'com.bitwarden:passwordless-android:1.0.1'
+```
+
+:::
+
+## Permissions
+
+In your `AndroidManifest.xml`, you will need to add the following permissions:
+
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+```
+
+## Configuration (Android application)
+
```

Comment:
Does it make sense to preface this with information about where to add this data class (e.g. like you do when you say "In your application's AndroidManifest.xml")?